README.md

```diff
@@ -53,13 +53,13 @@ These module automates the provisioning of components for running Oracle Databas
 You must [authenticate to your oci tenancy](https://docs.oracle.com/en-us/iaas/tools/oci-cli/3.43.1/oci_cli_docs/cmdref/session/authenticate.html) with config auth profile as `<MY_PROFILE_NAME>`. All available OCI regions are defined in [Regions and Availability Domains](https://docs.oracle.com/en-us/iaas/Content/General/Concepts/regions.htm#top).
 
 ``` shell
-oci session authenticate --region=<MY_REGION_IDENTIFIERr> --profile_name=<MY_PROFILE_NAME>
+oci session authenticate --region=<MY_REGION_IDENTIFIERr> --profile-name=<MY_PROFILE_NAME>
```

Comment:
is there an extra `r` in there? `MY_REGION_IDENTIFIERr>`

Reply:
fixed and thx.